lua/lazyvim/config/autocmds.lua

```diff
@@ -55,6 +55,7 @@ vim.api.nvim_create_autocmd("FileType", {
   group = augroup("close_with_q"),
   pattern = {
     "PlenaryTestPopup",
+    "grug-far",
```

Comment:
Maybe you would like to remove `spectre_panel` if you go on with removing `spectre` as a whole. Or maybe just leave it if you plan to move it to Extras. Just a suggestion, don't really know what you plan to do in the end. And if you do decide to remove it, maybe clean up `tsplayground` as well, as that's invalid with `Inspect Tree`.